crates/noirc_driver/src/lib.rs

```diff
@@ -305,5 +305,10 @@ pub fn compile_no_check(
     let (circuit, debug, abi) =
         create_circuit(program, options.show_ssa, options.show_brillig, show_output)?;
 
+    if options.print_acir {
+        println!("Unoptimised ACIR for main:");
```

Comment:
```suggestion
        println!("Unoptimized ACIR for main:");
```

Reply:
Ah, thanks for this review. We actually have some issues with the unoptimized circuit being printed twice (leading to our wild goose chase for memop opcodes yesterday.)